cypress/integration/granules_spec.js

```diff
@@ -225,7 +225,9 @@ describe('Dashboard Granules Page', () => {
         .first().contains('li', 'Completed').contains('li', 7)
         .next().contains('li', 'Failed').contains('li', 2)
         .next().contains('li', 'Running').contains('li', 2);
-      cy.setDatepickerDropdown('Recent');
+      // cy.setDatepickerDropdown('Recent');
```

Comment:
Why is this commented out?  If `recent` is selected, should the original numbers get displayed?
I think it's better to test both datepicker w and w/o granules returned.  Since 0 granules could mean issues with the query itself.

Reply:
Yeah that's a good point. Added to that test https://github.com/nasa/cumulus-dashboard/pull/812/commits/d551a5a0263e509e2d70554008281f3952c335db